baseapp-message-templates/baseapp_message_templates/email_utils.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from .models import EmailTemplate
+
+logger = logging.getLogger(__name__)
+
+
+def send_template_email(
```

Comment:
I think it would be better to have the args as
```
(template_name, *Args, **kwargs)
```
and then call the send method with
```
(*args, **kwargs)
```
so you don't have to manually redefine the args, and default values .